apps/sms/test/marionette/lib/views/conversation/view.js

```diff
@@ -7,13 +7,23 @@ var ConversationAccessor = require('./accessors');
 var appRoot = require('app-root-path');
 // TODO Change the path once requireFromApp becomes its own module
 var requireFromApp = require(appRoot +
-  '/shared/test/integration/require_from_app').requireFromApp;
+  '/shared/test/integration/require_from_app').require;
 
 function ConversationView(client) {
   this.client = client;
   this.accessors = new ConversationAccessor(client);
 }
 
+ConversationView.prototype = {
```

Comment:
Well, let's use either ```ConversationView.prototype = {``` or ```ConversationView.prototype.callContact = ``` notation, but not both. I'd prefer first, but would not object against second.